src/test/java/io/hexlet/typoreporter/service/TypoServiceIT.java

```diff
@@ -165,28 +165,28 @@ void deleteTypoByIdNotFound(final Long id) {
     @Test
     void getCountTypoByStatusForWorkspaceName() {
         List<Pair<TypoStatus, Long>> listWks = service.getCountTypoByStatusForWorkspaceName(WORKSPACE_101_NAME);
-        assertThat((long) listWks.size()).isEqualTo(4L);
+        assertThat(listWks.size()).isEqualTo(4);
```

Comment:
Don't use hardcoded values. Try to find this value from DB, you can use `findAll` method from repository.